litmus-portal/frontend/src/views/ChaosWorkflows/BrowseWorkflow/TableData.tsx

```diff
@@ -77,40 +117,84 @@ const TableData: React.FC<TableDataProps> = ({ data, exeData }) => {
           {nameCapitalized(data.cluster_name)}
         </Typography>
       </TableCell>
-      <TableCell>
-        <div className={classes.reliabiltyData}>
-          {exeData.finishedAt.length === 0 ? (
-            <Typography>
-              {t('workflowDetails.overallRR')}
-              <span className={classes.failed}>NA</span>
-            </Typography>
-          ) : exeData.phase === 'Failed' || exeData.phase === '' ? (
-            <>
-              <Typography>
-                {t('workflowDetails.overallRR')}
-                <span className={classes.failed}>0%</span>
-              </Typography>
-              <div className={classes.progressBar}>
-                <LinearProgressBar width={0.1} value={0} />
-              </div>
-            </>
+      <TableCell className={classes.reliabiltyData}>
+        <Typography>
+          <span>Overall RR: </span>
+          {exeData.resiliency_score === undefined ? (
+            <span className={classes.less}>NA</span>
           ) : (
-            <>
-              <Typography>
-                {t('workflowDetails.overallRR')}
-                <span className={classes.success}>100%</span>
-              </Typography>
-              <div className={classes.progressBar}>
-                <LinearProgressBar width={0.1} value={10} />
-              </div>
-            </>
+            <span className={getResiliencyScoreColor(exeData.resiliency_score)}>
+              <strong>{exeData.resiliency_score}%</strong>
+            </span>
           )}
-        </div>
+        </Typography>
+        <Typography>
+          <span>Experiments Passed: </span>
+          {exeData.resiliency_score === undefined ? (
+            <span className={classes.less}>NA</span>
+          ) : (
+            <span className={getResiliencyScoreColor(exeData.resiliency_score)}>
+              <strong>
+                {exeData.experiments_passed}/{exeData.total_experiments}
+              </strong>
+            </span>
```

Comment:
translations, fontWeight and the check can be reduced to !exeData.resiliency_score instead of exeData.resiliency_score === undefined